Mathlib/RingTheory/Finiteness.lean

```diff
@@ -565,11 +565,15 @@ theorem iff_addMonoid_fg {M : Type*} [AddCommMonoid M] : Module.Finite ℕ M ↔
     finite_def.2 <| (Submodule.fg_iff_addSubmonoid_fg ⊤).2 (AddMonoid.fg_def.1 h)⟩
 #align module.finite.iff_add_monoid_fg Module.Finite.iff_addMonoid_fg
 
+instance {M} [AddCommMonoid M] [AddMonoid.FG M] : Module.Finite ℕ M := iff_addMonoid_fg.mpr ‹_›
```

Comment:
Maybe this need not be an instance as I imagine it will get used very rarely if at all ...